NMDSdevopsServiceAdm/DataEngineering
# Create new cqc api provider schema

# Description
Add new provider schema, ordered correctly and abstracted to a data class as standardised.
Currently waiting on successful tests and deploy to test in branch and then on Athena

# How to test
Run the crawler after a job run and scan Athena to make sure the columns are in the correct order and not missing.

# Developer checklist
